streampipes-model/src/main/java/org/apache/streampipes/model/schema/EventProperty.java

```diff
@@ -30,9 +31,7 @@
 
 import javax.persistence.*;
 import java.net.URI;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.UUID;
+import java.util.*;
```

Comment:
Shall we use single imports, instead of `import *`

Reply:
I have removed java.util.* and added single imports.